zetaclient/chains/evm/client/client_live_test.go

```diff
@@ -41,6 +47,137 @@ func TestLiveClient(t *testing.T) {
 		})
 	})
 
+	t.Run("BlockByNumberCustom", func(t *testing.T) {
+		for _, tt := range []struct {
+			name        string
+			endpoint    string
+			block       uint64
+			checkHeader bool
+			assert      func(t *testing.T, v1 *geth.Block, errV1 error, v2 *Block, errV2 error)
+		}{
+			{
+				name:     "Both work for ETH",
```

Comment:
smaller case for the name might be better, "both work for ETH" or "both work for Ethereum"